addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties

```diff
@@ -0,0 +1,34 @@
+# Binding
+icloud.binding.name=Extension iCloud
+icloud.binding.description=Apple iCloud est utilis� pour obtenir des donn�es telles que le niveau de la batterie ou la localisation d'un ou plusieurs appareils Apple connect�s � un compte iCloud. Les mises � jour sont rapides et pr�cises sans impact notable sur la batterie.
+
+# Account Thing
+icloud.account-thing.label=Compte iCloud
```

Comment:
GitHub renders this like cr*p I assume you tested it and it works fine?

Reply:
Properties files can only be ISO-8859-1 per spec AFAIK, that's probably the reason.